profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -105,4 +108,33 @@ public static boolean getIncludeJvmInternalStacks(Config config) {
   public static boolean getTracingStacksOnly(Config config) {
     return config.getBoolean(CONFIG_KEY_TRACING_STACKS_ONLY, DEFAULT_TRACING_STACKS_ONLY);
   }
+
+  public static DataFormat getCpuDataFormat(Config config) {
+    String value = config.getString(CONFIG_KEY_CPU_DATA_FORMAT, "text");
+    return getDataFormat(value);
+  }
+
+  public static DataFormat getAllocationDataFormat(Config config) {
+    String value = config.getString(CONFIG_KEY_MEMORY_DATA_FORMAT, "pprof-gzip-base64");
```

Comment:
To switch CPU profiling to pprof by default, we need to enable this support on the backend in all realms. This will take some time. No such limitation for memory profiling data. We will make a separate request to switch the default as soon as we're confident that the backend is ready.